src/App.vue

```diff
@@ -1,85 +1,136 @@
+        },
+        {
+          'title': 'title 3',
+          'desc': 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Adipisci molestias quisquam culpa, iste pariatur ullam ab, accusamus id deserunt expedita dicta. Cumque ullam modi optio quibusdam dignissimos unde, animi officiis.'
+        },
+        {
+          'title': 'title 4',
+          'desc': 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Adipisci molestias quisquam culpa, iste pariatur ullam ab, accusamus id deserunt expedita dicta.'
+        },
+        {
+          'title': 'title 5',
+          'desc': 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Adipisci molestias quisquam culpa, iste pariatur ullam ab, accusamus id deserunt expedita dicta. Cumque ullam modi optio quibusdam dignissimos unde, animi officiis.'
+        },
+        {
+          'title': 'title 6',
+          'desc': 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Adipisci molestias quisquam culpa, iste pariatur ullam ab, accusamus id deserunt expedita dicta officiis.'
+        },
+        {
+          'title': 'title 7',
+          'desc': 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Adipisci molestias quisquam culpa, iste pariatur ullam ab, accusamus id deserunt expedita dicta. Cumque ullam modi optio quibusdam dignissimos unde, animi officiis.'
+        },
+      ],
+      filteredPosts: '',
+    }
+  },
+  methods: {
+    countWords(str) {
+      return str.trim().split(/\s+/).length;
+    },
+    loadData(){
+      this.filteredPosts = this.posts;
+      for (const post of this.filteredPosts) {
+        post.readActivated = false;
+      }
+      return this.filteredPosts;
+    }
+  },
+  mounted() {
+    this.loadData();
+  }
 }
-
-.logo {
-  display: block;
-  margin: 0 auto 2rem;
+</script>
+<!-- 
+<style>
+strong, i {
+  word-break: break-word;
 }
-
-nav {
-  width: 100%;
-  font-size: 12px;
+#app {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
   text-align: center;
-  margin-top: 2rem;
+  color: #2c3e50;
 }
-
-nav a.router-link-exact-active {
-  color: var(--color-text);
+.vue-logo {
+  width: 70px;
 }
-
-nav a.router-link-exact-active:hover {
-  background-color: transparent;
+.post-grid {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 20px;
+  list-style: none;
+  padding-left: 0;
 }
-
-nav a {
-  display: inline-block;
-  padding: 0 1rem;
-  border-left: 1px solid var(--color-border);
+.post-grid li {
+  border: 1px solid #ccc;
+  padding: 10px;
+  width: 300px;
+  border-radius: 4px;
+  box-shadow: 3px 3px 7px #ececec;
+  transition: all 0.27s ease-in-out;
 }
-
-nav a:first-of-type {
-  border: 0;
+.post-grid li:hover{
+  box-shadow: 3px 3px 7px #ccc;
```

Comment:
No `px` values only use `rem`.
Also, this can move into tailwind config so that it can be use later if required.